script.js

```diff
@@ -611,15 +615,16 @@ document.addEventListener("DOMContentLoaded", async () => {
       savedAlert.innerHTML += spanHtml;
 
       if (rateResult >= e.alertOn) {
-        savedAlert.innerHTML += `\n <span class="superAlert">⚠️alert⚠️<span> `;
+        // savedAlert.innerHTML += `\n <span class="superAlert">⚠️alert⚠️<span> `;
```

Comment:
It's better not to leave comments in the committed code. If you ever need to go back, you can always use the git history.